deploy/olm-catalog/portworx/2.0.0/core_v1_storagecluster_crd.yaml

```diff
@@ -55,14 +55,6 @@ spec:
                       storage cluster components. The key specifies component in format of "kind/component",
                       e.g. "deployment/stork" to pass custom annotations to stork deployment. The value is a map of
                       string that contains custom annotation key and value pairs.
-                  labels:
```

Comment:
Why remove it from the 2.0.0 spec? 2.0.0 reflects master and it should have the labels and preserveFullCustomImageRegistry changes.

Reply:
this was to revert all changes done by coping, it's removed because we when add the feature we forgot to make this change, we can review and update them later.